### PR TITLE
f-checkout@v1.4.0 - handle geolocation required error

### PR DESCRIPTION
**Pull request by FayeCarter**
### Changed
- `GEOLOCATION_REQUIRED` error handling.

### CODE REVIEW

File: packages/components/organisms/f-checkout/src/store/checkout.module.js [hunk 1]
```diff
@@ -483,6 +483,10 @@ export default {
                 state.address.administrativeArea = address.administrativeArea;
             }
 
+            if (fulfilment.location && fulfilment.location.geolocation) {
```
Comment (matthewhardern) [inline packages/components/organisms/f-checkout/src/store/checkout.module.js]: optional chaining would work here `fulfilment?.location?.geolocation`

Comment (matthewhardern) [reply packages/components/organisms/f-checkout/src/store/checkout.module.js]: `state.geolocation = fulfilment?.location?.geolocation || null;`

**Review by matthewhardern — APPROVED** [review]

**Review by AlixRichards — APPROVED** [review]

File: packages/components/organisms/f-checkout/src/tenants/en-AU.js [hunk 2]
```diff
@@ -128,6 +128,12 @@ const messages = {
                 buttonText: 'Ok'
             },
 
+            GEOLOCATION_REQUIRED: {
+                title: 'Something went wrong',
```
Comment (BanksySan) [inline packages/components/organisms/f-checkout/src/tenants/en-AU.js:132]: Is this the correct title?  _Something went wrong_ sounds a but unfriendly and many people (like my mum) wouldn't read past the title.

File: packages/components/organisms/f-checkout/src/tenants/en-GB.js [hunk 3]
```diff
@@ -118,6 +118,12 @@ const messages = {
                 buttonText: 'Ok'
             },
 
+            GEOLOCATION_REQUIRED: {
+                title: 'Something went wrong',
```
Comment (BanksySan) [inline packages/components/organisms/f-checkout/src/tenants/en-GB.js:122]: Ditto.

File: packages/components/organisms/f-checkout/src/tenants/en-NZ.js [hunk 4]
```diff
@@ -118,6 +118,12 @@ const messages = {
                 buttonText: 'Ok'
             },
 
+            GEOLOCATION_REQUIRED: {
+                title: 'Something went wrong',
```
Comment (BanksySan) [inline packages/components/organisms/f-checkout/src/tenants/en-NZ.js:122]: Ditto.

**Pull request merged**